src/main/java/org/folio/rest/impl/UserTenantsAPI.java

```diff
@@ -76,6 +76,20 @@ public void postUserTenants(String lang, UserTenant userTenant, Map<String, Stri
       });
   }
 
+  @Override
+  public void deleteUserTenants(Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    String okapiTenantId = TenantTool.tenantId(okapiHeaders);
+    userTenantService.deleteMemberUserTenant(okapiTenantId, vertxContext.owner())
```

Comment:
Current contract for `deleteMemberUserTenant` is that it successfully deletes exactly one record or throws an exception. Let's make it return Future<Void> then to avoid ambiguity. We can replace all 'boolean's with `void` in the chain.

Reply:
basically internally deleteMemberUserTenant invokes DAO method that also can return true or false, so we can not just simply return Future<Void> in this case, I also wrote java doc to explain contract more

Reply:
Can `deleteMemberUserTenant` return `false`?

Reply:
I think it can not. Do we handle a case if `deleteById` returns false?

Reply:
Added check for dao method as well